pgcopy/copy.py

```diff
@@ -326,13 +327,22 @@ def writestream(self, data, datastream):
         datastream.write(BINCOPY_TRAILER)
 
     def copystream(self, datastream):
-        columns = '", "'.join(self.cols)
-        cmd = 'COPY "{0}"."{1}" ("{2}") FROM STDIN WITH BINARY'
-        sql = cmd.format(self.schema, self.table, columns)
         cursor = self.conn.cursor()
+        table_name = psycopg2.sql.Identifier(
+            *(self.schema, self.table) if self.schema else (self.table,)
+        ).as_string(cursor)
+        columns = psycopg2.sql.SQL(", ") \
+            .join(
+                map(
+                    psycopg2.sql.Identifier,
+                    self.cols,
+                )
+            ) \
+            .as_string(cursor)
+        cmd = "COPY {} ({}) FROM STDIN WITH BINARY".format(table_name, columns)
```

Comment:
First off, thanks for your interest in using and improving pgcopy!

Proper handling for mixed case names would be a nice improvement for this library.  I want to make sure that we have test coverage for these scenarios.  I’ve updated the tests in [this branch](https://github.com/altaurog/pgcopy/tree/feat/case-sensitive-names).  Can you have a look?  Are there additional scenarios where it breaks?

The only change I had to make to get the tests to pass was in `util.py`.  Is this change necessary or just a preferred way of quoting?

Reply:
Both changes are necessary. I was getting this error:
```
psycopg2.errors.UndefinedTable: relation "casenamefff" does not exist
LINE 5:         WHERE c.oid = 'CaseNameFff'::regclass
```

Still debugging a different issue, but I'll create an issue on your repo about that.

Reply:
I the change in `util.py` addresses that error.  I tested manually on my branch and it is working for me.  Can you provide a minimum working example that doesn’t work?

(Granted, using `sql.Identifier` consistently is probably a good idea.  I just want to get clear what is a refactor and what is baseline for the functionality.)

Reply:
FYI: I've been updating everything to use `psycopg2.sql` throughout your codebase. Still a WiP but I'm doubtful I'll be able to complete until the weekend so feel free to carry on contributing to this PR (otherwise wait until then).

PS: Can we use `isort`, `black`, and `autoflake` in this repo? - The whitespace, quoting, line length, and import ordering is getting to me! :sweat_smile:

Reply:
Yes, `isort` and `black` are fine.  I use them on all my projects nowadays.  If you add them, please do it in a separate commit.